fulltext/__init__.py

```diff
@@ -151,38 +224,48 @@ def _get_file(backend, f, **kwargs):
             return backend._get_path(t.name, **kwargs)
 
 
+def get_backend_mod(ext):
+    if not ext.startswith('.'):
+        ext = "." + ext
+    try:
+        mime = EXTS_TO_MIMETYPES[ext]
+    except KeyError:
+        warn("don't know how to handle %r extension; assume binary" % ext)
+        mime = 'application/octet-stream'
+    mod_name = MIMETYPE_TO_BACKENDS[mime]
```

Comment:
Is it possible for `mime` to not be contained in `MIMETYPE_TO_BACKEND`?

Reply:
do you mean if this will ever raise `KeyError`? No, it is catched above and forced to 'application/octet-stream' which is always present

Reply:
That is what I meant. I guess I did not read closely enough to answer for myself.